catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -443,4 +443,12 @@
         <bundle>mvn:org.apache.mina/mina-core/${mina.version}</bundle>
     </feature>
 
+    <feature name="catalog-async-inmemory" install="auto" version="${project.version}">
```

Comment:
The reason I set it as auto is because Alliance will be using this and their imaging-app is auto install, so it would require this to be available on startup.